<ui version="4.0" >
 <class>StatusButtonClass</class>
 <widget class="QToolButton" name="StatusButtonClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>173</width>
    <height>34</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize" >
   <size>
    <width>16777215</width>
    <height>34</height>
   </size>
  </property>
  <property name="focusPolicy" >
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="popupMode" >
   <enum>QToolButton::InstantPopup</enum>
  </property>
  <property name="arrowType" >
   <enum>Qt::NoArrow</enum>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" >
   <item>
    <spacer name="horizontalSpacer_2" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>8</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="iconLabel" >
     <property name="maximumSize" >
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="text" >
      <string>TextLabel_DT</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="textLabel" >
     <property name="text" >
      <string>TextLabel_DT</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="arrowLabel" >
     <property name="text" >
      <string>TextLabel_DT</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
  <zorder>iconLabel</zorder>
  <zorder>textLabel</zorder>
  <zorder>arrowLabel</zorder>
 </widget>
 <layoutdefault spacing="6" margin="0" />
 <resources/>
 <connections/>
</ui>
